lib/main_common.pm

```diff
@@ -239,8 +239,14 @@ sub replace_opensuse_repos_tests {
     loadtest "console/zypper_ar";
 }
 
+sub is_updates_tests {
+    my $flavor = get_required_var('FLAVOR');
+    # Incidents might be also Incidents-Gnome or Incidents-Kernel
+    return $flavor =~ /Updates$/ || $flavor =~ /-Incidents/ || $flavor =~ /Maintenance$/;
```

Comment:
Please add a comment that the "-" is important and differs between openSUSE and SLE